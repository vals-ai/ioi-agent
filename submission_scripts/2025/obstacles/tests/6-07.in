version https://git-lfs.github.com/spec/v1
oid sha256:91a26ffeb26b6cb56f464435136af5f8a12a1b3cf09315cb74d2d075f8f9640b
size 7693724
