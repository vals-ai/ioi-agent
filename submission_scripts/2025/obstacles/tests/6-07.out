version https://git-lfs.github.com/spec/v1
oid sha256:b8a835d29e864f317691a5f88b007308e5198badad8ed70555419b213057eac4
size 400036
